### PR TITLE
Setup lint prettier

### PR DESCRIPTION
**Pull request by masbossun**
i think this is important as JS doesn't have any fix rules for codebases. But i don't know if you don't use this kind of tools. Just let me know if this tools not fit for you.

### CODE REVIEW

File: models/index.js [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+    return (
+      file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js'
+    );
   })
-  .forEach(file => {
-    const model = sequelize['import'](path.join(__dirname, file));
+  .forEach((file) => {
+    const model = sequelize.import(path.join(__dirname, file));
     db[model.name] = model;
   });
```
Comment (padulkemid) [inline models/index.js]: bekas generate dari sequelize diganti gpp mas bossun, tapi codenya jalan sih hehehe

**Comment by padulkemid** [comment]
bentar ya bossun, di cek dulu

**Comment by padulkemid** [comment]
SIPP, MERGEE!!!!

**Comment by padulkemid** [comment]
branchnya aku delete ya udh ga ada lagi kan yg harus diituin @masbossun

**Comment by masbossun** [comment]
Iya delete aja kalo feature branch udah di delete.

@padulkemid set aja config nya gimana, yang penting ada rules nya biar konsisten.

**Pull request merged**